### DISCUSSION TITLE
Feature Request: Option to disable/enable devices via Web UI

### DISCUSSION THREAD

**Discussion by juppyjupp** [Koenkk/zigbee2mqtt#16314] [General]
Hi community,

i have a question if it is possible to disable/enable configured devices vie the UI on demand.
Some of my Zigbee devices will be used only for a certain time of the year, e.g. the Christmas Tree plug in Dec./Jan. or the sprinkler in summer period.
If i simply unplug the corresponding devices from power, i will get frequently error messages this devices could not be found in the zigbee network.

The possibility to enable/disable devices would be an easy way to prevent those errors without the need of deleting or re-pairing.

WR Jupp

**Comment by juppyjupp** [comment]
Hi,

sorry after searching in discussions again with the right pattern i found:
[Feature Request: temporarily "disable" devices](https://github.com/Koenkk/zigbee2mqtt/issues/10003#top)
#10003 
is already handling this topic.

nevertheless it would be graet to have some kind of on/of switch in the web UI instead of changing the devices.yaml file manually.

Jupp

**Reply by Nardol** [reply]
It is already possible: in the Zigbee2MQTT dashboard click the device you want to disable, then go to settings, and check "disabled" then click save.
I use Zigbee2MQTT 1.29.2.

**Reply by juppyjupp** [reply]
Hi Nardol, all,

my fault ;-)
i missed to upgrade to latest version, was still running 1.28.
Feature was implmented in 1.29
After upgrading version i found the feature and disabled my "Christmas Tree" device and get rid of the ping failed messages.

Great support here, thanks.

Discussion can be closed

Jupp

**Reply by pohly** [reply]
I came here after searching for this feature. I have the "disabled" checkbox under settings, but with 1.39.0 I was not able to use it: whenever I set the checkmark, it gets unset before I can hit "Submit".

**Reply by Great-Chart** [reply]
I've also experienced this issue and don't believe it is related to OTA updates.
I have found that occasionally the Disabled checkbox would stay ticked long enough to scroll to the bottom of the page and hit Submit; (but not often)!
However, I have also discovered that (on Desktop implementation) ticking Disabled checkbox and then immediately hitting Enter key will apply the configuration change - with the red "Restart" button then becoming visible.

I'm not sure that the same workaround is practicable from within the mobile app.
Also commented in https://github.com/nurikk/zigbee2mqtt-frontend/issues/2135 - I presume this is the issue you have experienced too (that post includes a short video of the issue)

**Reply by cgatnet** [reply]
I've a similar experience. But on my side, the checkbox is removed only the first and second time, after I set it again, it remains set and I can submit it.

**Reply by justinhow** [reply]
I have the same issue with 1.40.2. With either the **Disable** or **Retain** check boxes whenever I set the checkmark, it gets unset before I can hit "Submit".

**Reply by gigagigagiga** [reply]
fyi: Works for me on Chrome but auto unticks itself under Safari
